sdk/openai/openai/test/utils/utils.ts

```diff
@@ -30,6 +30,7 @@ export enum APIVersion {
   Preview = "2025-01-01-preview",
   Stable = "2024-10-21",
   OpenAI = "OpenAI",
+  Realtime = "2024-10-01-preview",
```

Comment:
Could we rename this to the actual value?
```suggestion
  "2024_10_01_preview" = "2024-10-01-preview",
```

However, we will need to refer to it as `APIVersion ["2024_10_01_preview"]`